src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
```

Comment:
I think it wouldn't be a library then and we couldn't use the `using MarketExtensions for Market` pattern then though correct? I'm fine with either, just making sure that's the understanding.